src/web/src/controllers/Schedule.js

```diff
@@ -155,7 +155,15 @@ class Schedule {
       return false;
     }
   }
+  totalCredits() {
+    var array = Object.values(this.selectedCourses).map((c) => c.max_credits);
 
+    // Getting sum of numbers
+    var sum = array.reduce(function (a, b) {
+      return a + b;
+    }, 0);
+    return sum;
+  }
```

Comment:
Wyt about simplifying this function a tad bit? This way we don't define abstract sum function and use it once
```js
return Object.values(this.selectedCourses)
  .map((course) => course.max_credits)
  .reduce((sum, max_credits) => sum += max_credits)

```

Reply:
I think this is a good way to implement the function. I will definitely modify the function.